web_responsive/static/src/css/web_responsive.scss

```
@@ -160,6 +190,11 @@ $chatter_zone_width: 35%;
             display: flex;
             flex-direction: column;
             justify-content: flex-start;
+            background: none;
+            img {
+                box-shadow: none;
+                transition: 300ms all ease;
```

And you have to animate only the animated properties here. Notice that you're animating all, which includes the inherited `opacity` value, at a higher length that its parent (300ms vs. 100ms above). That's what produces the glitch from https://github.com/OCA/web/pull/1654#discussion_r466406100:

```suggestion
                transition: 300ms ease;
                transition-property: box-shadow, transform;
```

---

Thank you!, I missed that transition, I did not understand why it happened 😅

---

I think you forgot to include this change.